src/main/kotlin/no/nav/syfo/syfosmvarsel/nysykmelding/NySykmeldingService.kt

```diff
@@ -0,0 +1,66 @@
+package no.nav.syfo.syfosmvarsel.nysykmelding
+
+import no.nav.syfo.model.ReceivedSykmelding
+import no.nav.syfo.syfosmvarsel.LoggingMeta
+import no.nav.syfo.syfosmvarsel.domain.OppgaveVarsel
+import no.nav.syfo.syfosmvarsel.metrics.NY_SM_MOTTATT
+import no.nav.syfo.syfosmvarsel.metrics.NY_SM_VARSEL_OPPRETTET
+import no.nav.syfo.syfosmvarsel.util.innenforArbeidstidEllerPaafolgendeDag
+import no.nav.syfo.syfosmvarsel.varselutsending.VarselProducer
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+import java.util.*
+
+private val log: Logger = LoggerFactory.getLogger("no.nav.syfo.syfosmvarsel")
+
+// Henger sammen med tekster i mininnboks: http://stash.devillo.no/projects/FA/repos/mininnboks-tekster/browse/src/main/tekster/mininnboks/oppgavetekster
+const val OPPGAVETYPE = "0005"
+
+fun opprettVarselForNySykmelding(
+    receivedSykmelding: ReceivedSykmelding,
+    varselProducer: VarselProducer,
+    tjenesterUrl: String,
+    loggingMeta: LoggingMeta
+) {
+    try {
+        log.info("Mottatt ny sykmelding med id {}, $loggingMeta", receivedSykmelding.sykmelding.id, *loggingMeta.logValues)
+        NY_SM_MOTTATT.inc()
+
+        val oppgaveVarsel = receivedNySykmeldingTilOppgaveVarsel(receivedSykmelding, tjenesterUrl)
+        varselProducer.sendVarsel(oppgaveVarsel)
+        NY_SM_VARSEL_OPPRETTET.inc()
+        log.info("Opprettet oppgavevarsel for ny sykmelding med {}, $loggingMeta", receivedSykmelding.sykmelding.id, *loggingMeta.logValues)
+    } catch (e: Exception) {
+        log.error("Det skjedde en feil ved oppretting av varsel for ny sykmelding")
+        throw e
+    }
+}
+
+fun receivedNySykmeldingTilOppgaveVarsel(receivedSykmelding: ReceivedSykmelding, tjenesterUrl: String): OppgaveVarsel {
+    val utsendelsestidspunkt = LocalDateTime.now().innenforArbeidstidEllerPaafolgendeDag()
+    return OppgaveVarsel(
+            "NY_SYKMELDING",
```

Comment:
du må gjerne bruke labels her:
```
OppgaveVarsel(
    type = "NY_SYKMELDING",
    ressursId = ...
)
```
Da er det litt lettere å se hva de forskjellige feltene er for noe

Reply:
Ja, det var lurt!